microsoft-dotnet/3.1/modules/ROOT/pages/index.adoc

```diff
@@ -3,194 +3,50 @@
 
 Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
 
-Anypoint Connector for Microsoft .NET enables you to execute native .NET code in a Mule application.
+Anypoint Connector for Microsoft .NET (Microsoft .NET Connector) enables you to execute native .NET code in a Mule application.
 
-Supported scenarios include: executing code in an existing binary assembly (DLL) without modification, or new development of .NET code that extends the capability of your integration application.
+This connector executes .NET code using the Java Native Interface (JNI), which provides fast interprocess communication between the Java Virtual Machine (JVM) and a native application that runs in a host operating system.
```

Comment:
Omit articles before proper noun adjectives not followed by a common noun object:
```suggestion
This connector executes .NET code using Java Native Interface (JNI), which provides fast interprocess communication between Java Virtual Machine (JVM) and a native application that runs in a host operating system.
```